lib/server/routes/users.js

```diff
@@ -459,15 +459,15 @@ UsersRouter.prototype.confirmDestroyUser = function (req, res, next) {
       return next(new errors.NotFoundError('User not found'));
     }
 
-    Bucket.remove(
+    Bucket.deleteOne(
```

Comment:
This is a `deleteMany`, as we want to remove all the buckets associated with a user